src/Turbo/README.md

```diff
@@ -409,19 +410,23 @@ are passed to the template as variables: `entity`, `action` and `options`.
 
 ### Broadcast Conventions and Configuration
 
-The entity class **must** have a public property named `id` or a public method named `getId()`.
+When using the Mercure transport, the entity class **must** have a public property named `id` or a public method named `getId()`.
 
 If your entities aren't in the `App\Entity` namespace, Symfony UX Turbo will look for a template in a directory named after
 their Fully Qualified Class Names. For instance, if a class marked with the `Broadcast` attribute is named `\App\Foo\Bar`,
 the corresponding template will be `templates/broadcast/App/Foo/Bar.stream.html.twig`.
+// do we really need the above convention? Can't we force such entities to have a template attribute instead?
```

Comment:
I like to not have to care about the template name. Also, this convention is entirely optional and improves the DX a lot. Turbo Rails has a similar convention by the way.